src-plugins/iterativeClosestPoint/iterativeClosestPointPlugin.cpp

```diff
@@ -15,36 +15,6 @@
 #include "iterativeClosestPointPlugin.h"
 #include "iterativeClosestPointToolBox.h"
 
-#include <dtkLog/dtkLog.h>
-
-// /////////////////////////////////////////////////////////////////
-// iterativeClosestPointPluginPrivate
-// /////////////////////////////////////////////////////////////////
-
-class iterativeClosestPointPluginPrivate
-{
-public:
-    // Class variables go here.
-    static const char *s_Name;
-};
-
-const char * iterativeClosestPointPluginPrivate::s_Name = "iterativeClosestPoint";
-
-// /////////////////////////////////////////////////////////////////
-// iterativeClosestPointPlugin
-// /////////////////////////////////////////////////////////////////
-
-iterativeClosestPointPlugin::iterativeClosestPointPlugin(QObject *parent) : dtkPlugin(parent), d(new iterativeClosestPointPluginPrivate)
```

Comment:
In many of the plugin classes you have removed the parenting constructor, which results in one group of classes with that constructor and one group without it. I think it should be kept as all QObject subclasses should be "parentable".